minds/minds.py

```diff
@@ -37,6 +37,15 @@ def __init__(
 
         self.datasources = datasources
 
+    def __repr__(self):
+        return (f'Mind(name={self.name}, '
+                f'model_name={self.model_name}, '
+                f'provider={self.provider}, '
+                f'created_at={self.created_at}, '
```

Comment:
Maybe is better to present dates in 'yyyy-MM-dd HH:mm:ss' format?

Reply:
This crossed my mind but didn't implement it for the following reasons.

- Changing would be meddling with the API response. Hence, giving the response as is important. 
- I don't see the fields `created_at` and `updated_at` being used anywhere. 
- As an end user, the current format is easy to read the dates.

Reply:
1. what is the 'meddling with the API response' means? It is just way to format date to string. If format is not specified it will be formatted to string anyway but with default format.

3. for me it doesn't look good: it seems that provider is openai and created_at is Tue:
![image](https://github.com/user-attachments/assets/bcbb616d-9e3a-4e59-bd53-fbb0d70cc753)

Reply:
Sorry, I checked the response from server and understand what you mentioned. It is returned not in iso format. 
I think is better to update rest api and change format (to '2024-10-08T12:44:37Z') to make parsing of dates more save. Then SDK will store it as datetime type. And repr will convert it to string.

What we can in SDK at the moment (any option) in repr function::
- show dates  in quotes in the same format they came from server
- Or remove dates from repr for now

Reply:
Let's show the dates but in quotes. Changes done. 
![Screenshot 2024-10-09 at 8 04 16 PM](https://github.com/user-attachments/assets/10f9162f-ce19-4c1e-850d-d1677122e85b)